docs/examples/ead4d875877d618594d0cdbdd9b7998b.asciidoc

```diff
@@ -0,0 +1,6 @@
+// setup/add-nodes.asciidoc:170
+
+[source, python]
+----
+
```

Comment:
Any idea why this one is empty?

Reply:
@pquentin I was meaning to look into this more and totally spaced it (this is was a friday night fun project. I know 😂 ) I'll dig a bit more. I know Some of the json doc examples were strings instead of array bodies and was messing with the php examples I've been working on, but I assume that would be accommodated for already.

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/add-elasticsearch-nodes.html looks like the ruby examples only generated 1 of the 3 as well.

Reply:
I can confirm that our parser fails, maybe because of the comments, or because of the two different commands. Thanks, I'll go ahead and merge.